pkg/aspect/quotasManager.go

```diff
@@ -117,7 +117,7 @@ func (m *quotasManager) NewAspect(c *cpb.Combined, a adapter.Builder, env adapte
 
 func (*quotasManager) Kind() Kind                         { return QuotasKind }
 func (*quotasManager) DefaultConfig() config.AspectParams { return &aconfig.QuotasParams{} }
-func (*quotasManager) ValidateConfig(config.AspectParams, descriptor.Finder) (ce *adapter.ConfigErrors) {
+func (*quotasManager) ValidateConfig(config.AspectParams, expr.Validator, descriptor.Finder) (ce *adapter.ConfigErrors) {
```

Comment:
Yes, I think you're correct, we'll have to give the `Finder` to `NewAspect`. I'm about to wade into that as I start implementing validation in the manager, so I'll probably add it to the `Builder` interface in my next PR.